types/unl-core/unl-core-tests.ts

```diff
@@ -1,21 +1,100 @@
-import * as LocationId from 'unl-core';
+import * as UnlCore from 'unl-core';
 
 // Encoding
-const atx_locationId: string = LocationId.encode(30.2672, -97.7431);
-const atx_locationId_p3: string = LocationId.encode(30.2672, -97.7431, 3);
+const atx_locationId: string = UnlCore.encode(30.2672, -97.7431);
+const atx_locationId_p3: string = UnlCore.encode(30.2672, -97.7431, 3);
 
 // Decoding
-const atx_latlong: LocationId.PointWithElevation = LocationId.decode(atx_locationId);
+const atx_latlong: UnlCore.PointWithElevation = UnlCore.decode(atx_locationId);
 
 // Bounds
-const atx_bounds: LocationId.BoundsWithElevation = LocationId.bounds(atx_locationId);
+const atx_bounds: UnlCore.BoundsWithElevation = UnlCore.bounds(atx_locationId);
 
 // Adjacent
-const atx_adj_cell1: string = LocationId.adjacent(atx_locationId, 'N');
+const atx_adj_cell1: string = UnlCore.adjacent(atx_locationId, 'N');
 
 // Neighbors
-const atx_neighbors: LocationId.Neighbours = LocationId.neighbours(atx_locationId);
+const atx_neighbors: UnlCore.Neighbours = UnlCore.neighbours(atx_locationId);
 const atx_adj_cell3: string = atx_neighbors.n;
 
 // Grid Lines
-const atx_gridLines: Array<[[number, number], [number, number]]> = LocationId.gridLines(atx_bounds);
+const atx_gridLines: Array<[[number, number], [number, number]]> = UnlCore.gridLines(atx_bounds);
+
+// Polyhash
+
+// Sample polygon
+const grachtenGordel = [
+    [52.3860252, 4.882822],
+    [52.3721408, 4.8751831],
+    [52.3628122, 4.8804188],
+    [52.3578851, 4.8902035],
+    [52.3596149, 4.9085712],
+    [52.3625501, 4.9202442],
+    [52.3665334, 4.9275398],
+    [52.3716691, 4.9126911],
+    [52.3758612, 4.9129486],
+    [52.3776427, 4.9120903],
+    [52.3860252, 4.882822],
+];
+
+/**
+ * [{
+ *     precision: 7,
+ *     data: [
+ *        'u176p0e', '3yyj',
+ *        'zh3',    '72',
+ *        'ey',    'uq',
+ *        '9bjh',    '3zvc',
+ *        'y9',    'z0',
+ *        '6p0e'
+ *     ]
+ * }]
+ */
+const polyGordel = UnlCore.toPolyhash(grachtenGordel, 7);
```

Comment:
we need to specify the type for polyGordel to validate the return type of the called method. Same for other tests.